arches/app/utils/data_management/resources/formats/csvfile.py

```diff
@@ -766,24 +747,21 @@ def transform_value(datatype, value, source, nodeid):
                             value = datatype_instance.transform_value_for_tile(value, **config)
                             errors = datatype_instance.validate(value, row_number=row_number, source=source, nodeid=nodeid)
                         except Exception as e:
-                            errors.append(
-                                {
-                                    "type": "ERROR",
-                                    "message": "datatype: {0} value: {1} {2} - {3}".format(
-                                        datatype_instance.datatype_model.classname,
-                                        value,
-                                        source,
-                                        str(e) + " or is not a prefLabel in the given collection.",
-                                    ),
-                                }
+                            logger.warn(
```

Comment:
I think changing this to a warning is good, but the message itself can be improved. When I get an error parsing a resource instance value, it reads like this:

```
datatype: ResourceInstanceListDataType value: ([{'resourceId': '4d1c5b38-2e05-486e-ab17-61cfcf9e93b1',
'ontologyProperty': 'http://www.cidoc-crm.org/cidoc-crm/P14_carried_out_by', 'resourceXresourceId':
'','inverseOntologyProperty': 'http://www.cidoc-crm.org/cidoc-crm/P14i_performed'}], [{'resourceId': '5c5d91c0-
f51e-47c7-8f43-632d05a30341', 'ontologyProperty': 'http://www.cidoc-crm.org/cidoc-crm/P14_carried_out_by',
 'resourceXresourceId': '', 'inverseOntologyProperty': 'http://www.cidoc-crm.org/cidoc-crm/P14i_performed'}]) on
 line 2 - tuple indices must be integers or slices, not str or is not a prefLabel in the given collection.

```
This is a bit unclear because mention of a collection and prefLabel can give the impression that the value was somehow being read as a concept value rather than a resource-instance value.

The error is caused because Arches cannot parse/read the value in the CSV file. For that reason it seems like the error message should be something like: `The following value could not be interpreted as a [datatype] value: [value]`, and the message`or is not a prefLabel in the given collection` should be moved to the `transform_value_for_tile` method of the concept datatype so that it only appears when the issue is concept related.